[BUG] Node version in E2E tests is outdated

Current E2E test runs are failing on the CI because Node version is lower than the expected one:
```
Run pnpm install
 ERR_PNPM_UNSUPPORTED_ENGINE  Unsupported environment (bad pnpm and/or Node.js version)

Your Node version is incompatible with "/home/runner/work/minc-extension/minc-extension/podman-desktop".

Expected version: >=24.0.0
Got: v22.21.1

This is happening because the package's manifest has an engines.node field specified.
To fix this issue, install the required Node version.
Error: Process completed with exit code 1.
```
Reference run: https://github.com/minc-org/minc-extension/actions/runs/20990144888/job/60333345793

Updating the version should fix the issue

--- a/.github/workflows/e2e-main.yaml
+++ b/.github/workflows/e2e-main.yaml
@@ -73,7 +73,7 @@ jobs:
 
       - uses: actions/setup-node@v5
         with:
-          node-version: 22
+          node-version: 24
 
       - uses: pnpm/action-setup@v4
         name: Install pnpm

--- a/.github/workflows/pr-check.yaml
+++ b/.github/workflows/pr-check.yaml
@@ -137,8 +137,8 @@ jobs:
 
       - uses: actions/setup-node@v6
         with:
-          node-version: 22
-      
+          node-version: 24
+
         # on windows, wsl kernel update before this
       - name: Update Podman
         uses: redhat-actions/podman-install@main

